js/app.jsx

```diff
@@ -861,8 +861,8 @@ React.render(
   <Routes>
     <Route handler={App}>
       <Route name="app" path="/" handler={BigData} ignoreScrollBehavior={true} />
-      <Route name="job" path="/job/:jobId" handler={Job} />
-      <Route name="logs" path="/job/:jobId/logs" handler={JobLogs} />
+      <Route name="job" path="/job/:jobId/?" handler={Job} />
+      <Route name="logs" path="/job/:jobId/logs/?" handler={JobLogs} />
```

Comment:
Can we do redirects to the non-slash URLs so that there's a single canonical URL? Can we turn on something in react-router to handle that for us?